examples/step-66/doc/intro.dox

```diff
@@ -0,0 +1,260 @@
+<br>
+
+<i>This program was contributed by Fabian Castelli.
```

Comment:
Do you need to mention @tjhei  here?

Reply:
I didn't really do much work for the tutorial, except giving advice and reading.